app/CMakeLists.txt

```diff
@@ -7,9 +7,6 @@
 cmake_minimum_required(VERSION 3.13.1)
 find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
 
-project(app LANGUAGES C VERSION 1.0.0)
```

Comment:
One comment: until now, one could have a module with N apps, each with their own version (using CMake). Maybe not relevant in practice, but worth mentioning.

Reply:
This was brought up in https://github.com/zephyrproject-rtos/zephyr/issues/56605 as a limitation of this method as opposed to the other method. Another limitation of this method as opposed to the other offered implementation is that you cannot override versions of applications unless you change the file in the application tree, so if you add a module to your manifest and include that module to someone else's application, you are unable to change the version that the built application shows as.